.github/ISSUE_TEMPLATE/PLIP.md

```diff
@@ -4,7 +4,6 @@ about: Plone Improvement Proposal
 title: ''
 labels: ''
 assignees: ''
-
```

Comment:
As rule of thumb, do not touch (an edit is present) a file that has nothing to do with the PR. The changes should contain only changes relevant to the current matter.